React/README.md

```diff
@@ -21,11 +21,12 @@
     - [Exercise: controlled forms](exercises/forms-worksheet.md)
 1.  [Advanced Forms](forms-advanced.md)
 1.  [Testing](testing.md)
```

Comment:
I'm so confused! Haha. It looks like this link has been here for 2 years linking to nothing. Is that true? Or had the old react testing lesson been deleted already?

Reply:
It had been removed as I was working on week 2.